src/components/ModificationButton.js

```diff
@@ -3,37 +3,38 @@ import PropTypes from 'prop-types';
 
 const ModificationButton = ({ attName, buttonClass, maxPoints, pointsLeft, modificationFunction, attType, symbol }) => {
 	return (
-		<td>
-			<button
-				className={`btn ${buttonClass}`}
-				onClick={() => {
-					if (pointsLeft > 0) {
-						modificationFunction({
-							attribute: attName,
-							max: maxPoints,
-							spend: attType
-						});
-					}
-				}}
+		<button
+			className={`btn ${buttonClass}`}
+			onClick={() => {
+				if (pointsLeft === undefined || pointsLeft > 0) {
+					modificationFunction({
+						attribute: attName,
+						max: maxPoints,
+						spend: attType
+					});
+				}
+			}}
 			>
-				{symbol}
-			</button>
-		</td>
+			{symbol}
+		</button>
 	);
 };
 
 ModificationButton.propTypes = {
-	attName: PropTypes.string.isRequired,
+	attName: PropTypes.string,
 	buttonClass: PropTypes.string.isRequired,
-	maxPoints: PropTypes.number.isRequired,
+	maxPoints: PropTypes.number,
 	pointsLeft: PropTypes.number,
 	modificationFunction: PropTypes.func.isRequired,
-	attType: PropTypes.string.isRequired,
+	attType: PropTypes.string,
 	symbol: PropTypes.string.isRequired
 };
 
 ModificationButton.defaultProps = {
-	pointsLeft: 1
+	pointsLeft: 1,
+	attName: '!!!!!!!!!!!!!!!!!!!!',
```

Comment:
Aren't `attName`, `maxPoints`, and `attType` all required? I can't think of a time we wouldn't have them defined.

Reply:
You don't need them if you pass in a function that knows it already, is a subtracting button, and/or isn't limited by type.